spec/exchanges/dydx_perpetual/integration/market_spec.rb

```diff
@@ -13,17 +13,24 @@
     expect(pair.base).to_not be nil
     expect(pair.target).to_not be nil
     expect(pair.market).to eq market
+    expect(pair.inst_id).to eq "PBTC-USDC"
```

Comment:
also use `btc_usdc_pair.inst_id` and `btc_usdc_pair.contract_interval` to compare the data